src/Symfony/Component/Serializer/Tests/Normalizer/AbstractNormalizerTest.php

```diff
@@ -114,8 +114,18 @@ public function testGetAllowedAttributesAsObjects()
 
     public function testObjectWithStaticConstructor()
     {
-        $normalizer = new StaticConstructorNormalizer();
```

Comment:
To be sure that we don't introduce BC breaks, we prefer to not change existing tests but to introduce new ones.

Reply:
Done ! I will add some comments onto the compatibility layer I had to make ~